apps/showcase-e2e/src/integration/cypress-indexedDB-helper.spec.ts

```diff
@@ -1,87 +1,111 @@
+    });
 
-    cy.get('#firstName').should('be.visible').type('Hans');
-    cy.get('#lastName').should('be.visible').type('Gruber');
-    cy.get('#country').should('be.visible').type('Germany');
-    cy.get('#city').should('be.visible').type('Berlin');
+    it(`entering data into the form saves it to the indexedDb`, () => {
+      cy.visit('/cypress-helpers');
+
+      cy.get('#firstName').should('be.visible').type('Hans');
+      cy.get('#lastName').should('be.visible').type('Gruber');
+      cy.get('#country').should('be.visible').type('Germany');
+      cy.get('#city').should('be.visible').type('Berlin');
+
+      // eslint-disable-next-line cypress/no-unnecessary-waiting
+      cy.log('Waiting for the debounceTime to start a db write').wait(1100);
+
+      cy.getStore('@objectStore').readItem('user_form').should('deep.equal', {
+        firstName: 'Hans',
+        lastName: 'Gruber',
+        country: 'Germany',
+        city: 'Berlin',
+        address: '',
+        addressOptional: '',
+      });
+    });
 
-    // eslint-disable-next-line cypress/no-unnecessary-waiting
-    cy.log('Waiting for the debounceTime to start a db write').wait(1100);
+    it(`when the indexedDb is deleted manually and then the page reloaded, the form does not populate`, () => {
+      cy.visit('/cypress-helpers');
 
-    cy.getStore('@objectStore').readItem('user_form').should('deep.equal', {
-      firstName: 'Hans',
-      lastName: 'Gruber',
-      country: 'Germany',
-      city: 'Berlin',
-      address: '',
-      addressOptional: '',
-    });
-  });
+      cy.get('#firstName').should('be.visible').type('Hans');
+      cy.get('#lastName').should('be.visible').type('Gruber');
+      cy.get('#country').should('be.visible').type('Germany');
+      cy.get('#city').should('be.visible').type('Berlin');
 
-  it(`when the indexedDb is deleted manually and then the page reloaded, the form does not populate`, () => {
-    cy.visit('/cypress-helpers');
+      // eslint-disable-next-line cypress/no-unnecessary-waiting
+      cy.log('Waiting for the debounceTime to start a db write').wait(1100);
 
-    cy.get('#firstName').should('be.visible').type('Hans');
-    cy.get('#lastName').should('be.visible').type('Gruber');
-    cy.get('#country').should('be.visible').type('Germany');
-    cy.get('#city').should('be.visible').type('Berlin');
+      cy.log('user manually clears the IndexedDb')
+        .getStore('@objectStore')
+        .deleteItem('user_form')
+        .keys()
+        .should('have.length', 0);
+      cy.reload();
 
-    // eslint-disable-next-line cypress/no-unnecessary-waiting
-    cy.log('Waiting for the debounceTime to start a db write').wait(1100);
+      cy.get('#firstName').should('be.visible').and('have.value', '');
+      cy.get('#lastName').should('be.visible').and('have.value', '');
+      cy.get('#country').should('be.visible').and('have.value', '');
+      cy.get('#city').should('be.visible').and('have.value', '');
```

Comment:
In a similar way to my previous suggestion:

```ts
Cypress.Commands.add('visibleAndEmpty', (id) => {
  cy.get(id).should('be.visible').and('have.value', '');
})
```

And then use it as:

```ts
cy.visibleAndEmpty('#firstName');
cy.visibleAndEmpty('#lastName');
```

or even better, a utility command to send all Ids at once:

```ts
cy.visibleAndEmpty('#firstName', '#lastName', '#country', '#city');
```